docs/open-source-project/release-notes/tizen-5-5-m1.md

```diff
@@ -0,0 +1,631 @@
+    - API set to get EAP anonymous identity has been added.
+  - Bluetooth
+    - Privilege level of OOB data getter API has been changed.
+  - Smart traffic control
+    - API to clone and destroy statistics info handle has been added.
+    - Legacy API to get statistics info asynchronously has been deprecated.
+    - API to get all statistics info has been added.
+    - Legacy API to get total statistics info has been deprecated.
+  - curl version up (7.62.0)
+  - C# API
+    - BluetoothLeDevice API set getting the remote device information has been added.
+    - Legacy C# Gatt API has been deprecated.
+    - New BluetoothGattClient C# API has been added.
+    - BluetoothLeDevice API set getting the remote device information has been deprecated.
+    - Bluetooth - Wrong AcceptStateChangedEventArgs class property and DestroyServerSocket API has been deprecated.
+    - Bluetooth - New SocketConnection class property has been added.
+    - Bluetooth - API to destroy Gatt server has been added.
+    - STC C# API has been added.
+    - API to check Wi-Fi scan state has been added.
+    - API to check cellular roaming state has been added.
+    - Wi-Fi - Exceptions to provide accurate error information has been added.
+
+
+## Security
+
+### New and changed features
+
+- Security-manager
+  - Smack rules will be loaded directly from security-manager DB; all rule files have been removed.
+- Privilege
+  - The following privileges have been added.
+    - autofillmanager, internal/buxton/systemsetting, filesystem.read, filesystem.write, windowsystem.admin, d2d.datasharing, d2d.remotelaunch
+- Privilege-info
+  - New API has been added.
+    - Getting several privilege information as list.
+
+
+## Service framework
+
+### New and changed features
+
+- Account -Manager
+  - Added feature to query account information even after account deletion.
+
+### Fixes
+
+- Account-Manager
+  - Removed unwanted D-Bus API calls.
+  - Fixed Memory Leaks.
+  - Fix to remove provider information from global account db.
+- Sync-Manager
+  - Enabled Additional D-Bus error handling.
+  - Handled potential defects.
+- Calendar-service
+  - Supported calendar DB purge when Device is reset.
+  - Fixed potential defects.
+- Email-service
+  - Fixed Potential defects.
+- Message Service
+  - Fixed Potential defects.
+- Location Service
+  - Handled potential defects.
+  - Replaced deprecated capi-network-wifi dependency.
+- Sensord
+  - Fixed potential defects.
+  - Fixed timestamp data type issue.
+- Phonenumber-Utils
+  - Fixed Potential defects.
+
+
+## Web framework
+
+### New and changed features
+
+- Web Engine
+  - Web Engine Upgrade Framework has been added.
+    - chromium-efl package can be upgraded with TPK, not whole image upgrade.
+    - chromium-efl-squashfs package has been added to provide the preload web engine as a squashfs image file, not regular file.
+    - chromium-efl squashfs image file is compressed by about 60%.
```

Comment:
Suggest rephrasing to:

chromium-efl squashfs image file has been compressed by about 60%.

Reply:
will accept